AegisBuild.mk

```diff
@@ -16,22 +16,38 @@ build: \
 	example-multiple-secrets-push \
 	example-init-container-bundle \
 	example-init-container-push \
-	safe-bundle \
-	safe-push \
+	safe-bundle-ist \
+	safe-push-ist \
+	safe-bundle-ist-fips \
```

Comment:
Being explicit and suffixing them with -ist;

Also one more command for fips-compliant images.